src/vs/workbench/contrib/notebook/browser/diff/notebookDiffActions.ts

```diff
@@ -741,7 +742,7 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 		},
 		'notebook.diff.experimental.toggleInline': {
 			type: 'boolean',
-			default: false,
+			default: typeof product.quality === 'string' && product.quality !== 'stable', // only enable as default in insiders
```

Comment:
`product` is deprecated, hence you might want https://github.com/microsoft/vscode/pull/241042

Reply:
If you think we can ignore that, happy to approve, else I think we should change this.
Let me know.

Reply:
We typically prefer to use `IProductService` if possible but for temporary setting overrides, direct use of `product` is fine.